dot/parachain/candidate-validation/candidate_validation.go

```diff
@@ -38,25 +43,16 @@ type BlockState interface {
 func NewCandidateValidation(overseerChan chan<- any, blockState BlockState) *CandidateValidation {
 	candidateValidation := CandidateValidation{
 		SubsystemToOverseer: overseerChan,
+		pvfHost:             pvf.NewValidationHost(),
 		BlockState:          blockState,
 	}
 	return &candidateValidation
 }
 
 // Run starts the CandidateValidation subsystem
-func (cv *CandidateValidation) Run(ctx context.Context, overseerToSubsystem <-chan any) {
-	for {
-		select {
-		case msg := <-overseerToSubsystem:
-			logger.Debugf("received message %v", msg)
-			cv.processMessage(msg)
-		case <-ctx.Done():
-			if err := ctx.Err(); err != nil {
-				logger.Errorf("ctx error: %s\n", err)
-			}
-			return
-		}
-	}
+func (cv *CandidateValidation) Run(context.Context, <-chan any) {
+	cv.wg.Add(1)
+	go cv.processMessages(&cv.wg)
 }
```

Comment:
Why?

Reply:
I thought this looked cleaner, we discussed this so I've reverted it.